main.css

```diff
@@ -0,0 +1,634 @@
+  height: 75%;
+  border: 4px solid red;
+  border-radius: 5px;
+}
+
+.container {
+  width: 100%;
+  height: 700px;
+  margin: 0;
+  padding: 0;
+  background-color: rgb(45, 103, 151);
+}
+
+.container-article {
+  text-align: center;
+  color: white;
+}
+
+.container-article button i {
+  font-size: 0.8em;
+}
+
+.grid-4 {
+  display: grid;
+  grid-template-columns: 400px 1fr;
+  margin: 5% 0;
+}
+
+.box-img-md {
+  width: 200px;
+  height: 150px;
+  border: 4px solid #333;
+  border-radius: 10px;
+  margin-left: 10%;
+}
+
+.grid-4 p span {
+  text-decoration: underline;
+  color: rgb(44, 135, 165);
+  cursor: pointer;
+}
+
+.grid-5 {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  margin: 0 10%;
+  grid-row-gap: 150px;
+}
+
+.article-header-2 {
+  display: flex;
+}
+
+.article-header-2 h5 span {
+  text-decoration: none;
+  font-weight: lighter;
+  font-style: oblique;
+  color: #555;
+}
+
+.grid-5 p span {
+  font-weight: lighter;
+  color: #777;
+}
+
+.container-2 {
+  width: 100%;
+  height: auto;
+  margin: 10% 0 0 0;
+  padding: 0;
+  background-color: #d33a2c;
+  display: flex;
+}
+
+.container-article-2 {
+  color: white;
+}
+
+.box-img-lg {
+  width: 400px;
+  height: 300px;
+  border: 4px solid #333;
+  border-radius: 10px;
+  margin-right: 10%;
+  margin-top: 2%;
+}
+
+.grid-6 {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  margin: 10% 5%;
+  grid-column-gap: 20px;
+  grid-row-gap: 50px;
+}
+
+.grid-6 article {
+  background-color: rgb(240, 240, 240);
+  border: 0.5px solid white;
+  border-radius: 10px;
+}
+
+.js {
+  color: #19a972;
+}
+
+.css {
+  color: #ff5a00;
+}
+
+.web {
+  color: #aa1994;
+}
+
+.email {
+  color: #0168b8;
+}
+
+.js-btn {
+  background-color: #19a972;
+}
+
+.css-btn {
+  background-color: #ff5a00;
+}
+
+.web-btn {
+  background-color: #aa1994;
+}
+
+.email-btn {
+  background-color: #0168b8;
+}
+
+.high-article {
+  height: 900px;
+}
+
+.article-flex {
+  display: flex;
+  justify-content: center;
+}
+
+.article-flex input {
+  float: left;
+  padding: 5px 80px;
+  border-radius: 5px;
+  margin: 5% 0;
+  border: 0;
+  height: 70px;
+}
+
+.article-flex input::placeholder {
+  font-size: 1.8em;
+  color: #777;
+  font-style: oblique;
+  font-family: "Times New Roman", Times, serif;
+}
+
+.article-end {
+  background-color: #2b8cd6;
+  text-align: center;
+  width: 70%;
+  margin: 5% auto;
+  border-radius: 10px;
+  color: white;
+  font-family: "Times New Roman", Times, serif;
+  padding: 15px 5px;
+  font-size: 1.1em;
+}
+
+footer {
+  background-color: #d33a2c;
+  font-family: "Times New Roman", Times, serif;
+}
+
+footer h3 {
+  padding: 3% 0;
+  text-align: center;
+  color: white;
+  font-size: 22px;
+}
+
+footer .first-list {
+  display: flex;
+  margin: 5% auto;
+  flex-wrap: wrap;
+}
+
+.second-class li {
+  padding: 20px 0;
+  list-style: none;
+  margin: 20px auto;
+}
+
+footer .first-list li {
+  line-height: 40px;
+  margin: 0 15px;
+}
+
+.second-class {
+  display: flex;
+}
```

Comment:
This selector has the same styles of the selector on line 482. Minimize code by grouping selectors with the same style definition.